ablator/main/model/wrapper.py

```diff
@@ -607,6 +946,14 @@ def make_dataloaders(self, run_config: RunConfig) -> None:
         self.test_dataloader = self.make_dataloader_test(run_config)
 
     def checkpoint(self, is_best=False):
+        """
+        Save a checkpoint of the model.
```

Comment:
Specify how it's saved? i.e directory name (`best` if the trained model achieves the best performance, else `recent`), file name (is named after model class name + current iteration is included in the checkpoint name, etc.)

Reply:
I think details can be added in  logger, it just calls the function in logger